src/aircraft/FlyByWireA380X.ts

```diff
@@ -0,0 +1,141 @@
+import { AircraftConfigSimType, AircraftFeature, FeatureType } from '../defs'
+import {
+    AircraftConfig,
+    FeatureAddresses,
+    FeatureState,
+    FlapNames,
+    Meta,
+} from '../interface/aircraft'
+
+export default class FlyByWireA380X extends AircraftConfig {
+    meta: Meta = {
+        id: 'flybywire_a380x',
+        name: 'FlyByWire A380X',
+        sim: AircraftConfigSimType.MsFs,
+        enabled: true,
+        priority: 2,
+        author: 'Arthur Parienté <https://github.com/arthurpar06>'
+    }
+
+    features: FeatureAddresses = {
+        [AircraftFeature.BeaconLights]: {
+            LIGHTING_BEACON_0: FeatureType.Int,
+        },
+        [AircraftFeature.LandingLights]: {
+            LIGHTING_LANDING_2: FeatureType.Int,
+        },
+        [AircraftFeature.LogoLights]: {
+            // Lhe logo light isn't available in the FBW A380X at the moment; in the sim, the navigation light currently controls the logo ligh
```

Comment:
There is no `Logo Light` switch on Airbuses, technically it is automatic and tied to the Navigation Lights switch. Works with some pressure sensors on gear strut and altitude as far as I remember. So settings Nav Lights to 1 or 2 will be enough to have Logo Lights too 😉

Reply:
The A380 isn't like other Airbus models; it has a specific logo light, although it hasn't been implemented yet in the FBW.

![image](https://github.com/user-attachments/assets/6e7097d1-3dba-4dea-8190-705eebba70ff)

Reply:
If this is the case, then yes we should have it :)